domains/whoami.json

```diff
@@ -7,7 +7,7 @@
         },
     
         "record": {
-            "CNAME": "letsedit.cloudns.be"
+            "CNAME": "samir717le.github.io"
```

Comment:
Unable to view any contents of site. Provide a preview. Double check pull requests meet all requirements as mentioned in the pull request template.